lib/experimental/site-editor-permalinks.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+add_action(
+	'block_editor_settings_all',
+	function ( $settings ) {
+		$settings['__experimentalDashboardLink'] = admin_url();
+		return $settings;
```

Comment:
If this is an action, why do we need this callback to return anything? I think you mean to use `add_filter()` instead of `add_action()`. Also, do you mind declaring the function separately? That allows an easier way for someone to un-hook it if they want.

Reply:
Good catch it'a a filter